### PR TITLE
Cli arguments

### LINKED ISSUE #39: --noinput
Hi, when I run normal tests using `test` management command and test database already exists, command line asks me if I want existing database to be deleted, ex:
```
Creating test database for alias 'default'...
Got an error creating the test database: (1007, "Can't create database 'test_project'; database exists")
Type 'yes' if you would like to try deleting the test database 'test_project', or 'no' to cancel:
```

Same prompt can popup for `behave` command.

`test` management command (and many other) have --noinput parameter to suppress these prompts and always confirm/overwrite:
https://docs.djangoproject.com/en/1.11/ref/django-admin/#cmdoption-test-noinput

Would it be possible to add `--noinput` to `behave` command as well?

### PR DESCRIPTION
**Pull request by alex-hutton**
Resolves #39

### CODE REVIEW

**Comment by alex-hutton** [comment]
@bittner in regards to the tox failure, I get the same failure running against master when I run locally. It is a problem with the lack of existance of a `steps/` directory. If I creat an empty directory it passes but other problems occur later when the behave tests run. Do you have pointers?

**Comment by alex-hutton** [comment]
I noticed the same problem was affecting #57 , so I added the line https://github.com/behave/behave-django/pull/62/files#diff-b91f3d5bd63fcd17221b267e851608e8R20 .

**Comment by bittner** [comment]
As you figured the failing tests have something to do with changes in the latest release of *behave*. We eventually need to fix this, either here or in *behave*.

**Comment by bittner** [comment]
Can you extend the test matrix in [tox.ini](https://github.com/behave/behave-django/blob/master/tox.ini) and [.travis.yml](https://github.com/behave/behave-django/blob/master/.travis.yml)? This way we could test against the old and the current version of *behave* (and maybe even the repository HEAD). You can [allow tests to fail](https://docs.travis-ci.com/user/customizing-the-build/#Rows-that-are-Allowed-to-Fail) against the new version.

What wonders me is that the Python 3.3 + Django 1.8 combo fails. I can't see a clear reason, only a deprecation warning.

**Comment by bittner** [comment]
This issue with the `steps/` directory is fixed now via restructuring the directory tree. This means, you'll have to move (the changes of) your tests to the new file( location)s.

Can you to rebase your branch and/or redo the changes?

**Comment by bittner** [comment]
> Can you extend the test matrix in tox.ini and .travis.yml? This way we could test against the old and the current version of behave (and maybe even the repository HEAD). You can allow tests to fail against the new version.

This is provided by PR #70.

**Comment by alex-hutton** [comment]
Hello, I have merged in master and everything seems ok. However, master seems to have failing tests:

```shell
Failing scenarios:
  tests/acceptance/features/fixture-decorator.feature:7  Load fixtures with the decorator
  tests/acceptance/features/fixture-decorator.feature:11  Load multiple fixtures and callables
```

**Comment by bittner** [comment]
Thanks! Don't worry about the failing tests; they are related to issue #67. :roll_eyes: 

@mixxorz Is there anything you see problematic or worth improving with the contribution? (I recall, you had some comment on that we should look into a generic solution for passing through CLI options. Not sure where it was.) -- I would otherwise merge this PR.

**Comment by bittner** [comment]
@alex-hutton 
Would the additions be worth mentioning in the [Usage docs](https://github.com/behave/behave-django/blob/master/docs/usage.rst#command-line-options)?

Would you want to update the [AUTHORS](https://github.com/behave/behave-django/blob/master/AUTHORS.rst) and [HISTORY](https://github.com/behave/behave-django/blob/master/HISTORY.md), please?

**Comment by alex-hutton** [comment]
@bittner updated.

I rewrote the docs a little bit because it seemed to be warranted. I also used `1.2.0` as the next version in `HISTORY.rst`,  I assume this is what you wanted.

**Comment by bittner** [comment]
Could you rebase again now? We should then get a green build.

File: tests/unit/test_passthru_args.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+@mock.patch('behave_django.management.commands.behave.BehaviorDrivenTestRunner')  # noqa
+class TestPassThruArgs(DjangoSetupMixin):
+
+    def test_keepdb_flag(self,
+                         mock_test_runner,
+                         mock_behave_main):
+        """Test if keepdb is properly set on the test_runner."""
+
+        self.run_management_command('behave', keepdb=True)
+        assert mock_test_runner.call_args[1]['keepdb'] is True
```
Comment (bittner) [inline tests/unit/test_passthru_args.py]: Should we maybe write that like this? (in all occurrences)
```python
    args, kwargs = mock_test_runner.call_args
    assert kwargs['keepdb'] is True
```
To make it self-explanatory.

Comment (bittner) [reply tests/unit/test_passthru_args.py]: @alex-hutton Would you care to adapt the code as I proposed?

@mixxorz Any opinion on the whole PR as such? (See my [comment above](https://github.com/behave/behave-django/pull/62#issuecomment-443553325).)

**Comment by bittner** [comment]
>  But I wrote it slightly differently (because `args` is not used):

That's fine.

**Comment by alex-hutton** [comment]
@bittner , updated.

**Comment by bittner** [comment]
Thanks @alex-hutton!

**Pull request merged**